src/__main__.py

```diff
@@ -1,35 +1,89 @@
 import credentials
```

Comment:
Where are you using this script? Is it just for testing?

Reply:
when running on my machine for testing, I added my real `credentials.py` file into `.gitignore`